genienlp/data_utils/bootleg.py

```diff
@@ -293,7 +295,68 @@ def post_process_bootleg_types(qid, type, title, almond_domains):
             elif title in ['video game', 'disease', 'city of the United States', 'taxon',
                            'Wikimedia disambiguation page', 'Wikimedia list article']:
                 type = 'unk'
+
+        elif domain == 'mario':
```

Comment:
We should avoid using the name mario anywhere outside of Kubeflow. Nobody would understand what's going on.